### PR TITLE
ADD: Arabic language

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by Overtorment** [comment]
uploaded files to transifex

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/61jyyv7u8kjgv5zkk5490r8bz8

[download apk](http://filestorage.bluewallet.io:1488/1599661020.apk)

File: ios/fastlane/metadata/ar-SA/keywords.txt [hunk 1]
```diff
@@ -1 +1 @@
-bitcoin,wallet,bitcoin wallet,blockchain,btc,cryptocurrency,buy bitcoin,samourai,electrum,ethereum
```
Comment (limpbrains) [inline ios/fastlane/metadata/ar-SA/keywords.txt:1]: inverted commas o_0

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: ios/fastlane/metadata/ar-SA/keywords.txt (hunk 1 above)
Comment (Overtorment) [reply ios/fastlane/metadata/ar-SA/keywords.txt:1]: I fixed them to regular commas, but not sure if that is correct O_o

**Pull request merged**